documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -263,128 +235,104 @@ public void run() {
 
 ## ReaderGroup Basics
 
-Any Reader in Pravega belongs to some ReaderGroup.  A ReaderGroup is a grouping
-of one or more Readers that consume from a Stream in parallel.  Before we create
-a Reader, we need to either create a ReaderGroup (or be aware of the name of an
-existing ReaderGroup).  This application only uses the basics from ReaderGroup.
-
-Lines 10-15 show basic ReaderGroup creation.  ReaderGroup objects are created
-from a ReaderGroupManager object.  The ReaderGroupManager object, in turn, is
-created on a given Scope with a URI to one of the Pravega Controllers, very much
-like a ClientFactory is created.  A ReaderGroupManager object is created on line
-14.  Note the creation is also in a try-with-resources statement to make sure
-the ReaderGroupManager is properly cleaned up.   The ReaderGroupManager allows a
-developer to create, delete and retrieve ReaderGroup objects by name.
-
-To create a ReaderGroup, the developer needs a name for the ReaderGroup, a
-configuration with a set of 1 or more Streams to read from.  
-
-The ReaderGroup's name might be meaningful to the application, like
-"WebClickStreamReaders".  In our case, on line 10, we have a simple UUID as the
-name (note the modification of the UUID string to remove the "-" character
-because ReaderGroup names can only have letters and numbers).  In cases where
-you will have multiple Readers reading in parallel and each Reader in a separate
-process, it is helpful to have a human readable name for the ReaderGroup.  In
-our case, we have one Reader, reading in isolation, so a UUID is a safe way to
-name the ReaderGroup.  Since the ReaderGroup is created via the
-ReaderGroupManager and since the ReaderGroupManager is created within the
-context of a Scope, we can safely conclude that ReaderGroup names are namespaced
+Any Reader in Pravega belongs to some Reader Group.  A Reader Group is a grouping
+of one or more Readers that consume from a Stream in parallel. Before we create
+a Reader, we need to either create a Reader Group (or be aware of the name of an
+existing Reader Group). This application only uses the basics from Reader Group.
+
+Reader Group objects are created from a `ReaderGroupManager` object. The `ReaderGroupManager` object, in turn, is created on a given Scope with a URI to one of the Pravega Controllers, very much
+like a `ClientFactory` is created. Note that, the `createReaderGroup()` is also in a try-with-resources statement to make sure that the `ReaderGroupManager` is properly cleaned up. The `ReaderGroupManager` allows a developer to create, delete and retrieve Reader Group objects using the name.
+
+To create a Reader Group, the developer needs a name for the Reader Group, a
+configuration with a set of one or more Streams to read from. The Reader Group's name might be meaningful to the application, like
+"WebClickStreamReaders".  In our example ```UUID.randomUUID().toString().replace("-", "")``` we have a simple UUID as the
```

Comment:
I suggest to remove `(*note the modification of the UUID string to remove the "-" character
 because Reader Group names can only have letters and numbers*)`, as it only complicates things. The previous sentence  we explicitly say `alphanumeric`.

Reply:
Accepted Suggestion